unit/util/simplify_expr.cpp

```diff
@@ -516,3 +516,27 @@ TEST_CASE("Simplify bitor", "[core][util]")
     REQUIRE(simplify_expr(bitor_exprt{b, zero}, ns) == b);
   }
 }
+
+TEST_CASE("Simplify inequality", "[core][util]")
+{
+  symbol_tablet symbol_table;
+  namespacet ns(symbol_table);
+
+  {
+    // This checks that 3 < (B ? 4 : 5) simplifies to true, just like (B ? 4 :
+    // 5) > 5 simplifies to true.
```

Comment:
(B ? 4 : 5) > 5 simplifies to false?

Reply:
Thank you for catching this typo in the comment! (It should have said/now says "> 3")